Documentation/building/debugging-instructions.md

```diff
@@ -89,8 +94,8 @@ However the common commands have been aliased so that you don't need the SOS pre
     clrstack        -> sos ClrStack
     clrthreads      -> sos Threads
     dso             -> sos DumpStackObjects
-	dumpclass       -> sos DumpClass
-	dumpheap        -> sos DumpHeap
+    dumpclass       -> sos DumpClass
```

Comment:
Done.



Sent from my Verizon, Samsung Galaxy smartphone



-------- Original message --------
From: Jan Vorlicek <notifications@github.com>
Date: 6/6/16 12:26 PM (GMT-08:00)
To: dotnet/coreclr <coreclr@noreply.github.com>
Cc: Mike McLaughlin <mikem@microsoft.com>, Assign <assign@noreply.github.com>
Subject: Re: [dotnet/coreclr] Add doc on how to building lldb for OS X. (#5520)

-------- Original message --------
From: Jan Vorlicek <notifications@github.com>
Date: 6/6/16 12:26 PM (GMT-08:00)
To: dotnet/coreclr <coreclr@noreply.github.com>
Cc: Mike McLaughlin <mikem@microsoft.com>, Assign <assign@noreply.github.com>
Subject: Re: [dotnet/coreclr] Add doc on how to building lldb for OS X. (#5520)


In Documentation/building/debugging-instructions.md<https://github.com/dotnet/coreclr/pull/5520#discussion_r65952320>:

> @@ -89,8 +94,8 @@ However the common commands have been aliased so that you don't need the SOS pre
>      clrstack        -> sos ClrStack
>      clrthreads      -> sos Threads
>      dso             -> sos DumpStackObjects
> -     dumpclass       -> sos DumpClass
> -     dumpheap        -> sos DumpHeap
> +    dumpclass       -> sos DumpClass


The histXXX commands are missing here, could you please add them?

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub<https://github.com/dotnet/coreclr/pull/5520/files/4e2bda030779bcea237579fdefb893a8fe3c957d#r65952320>, or mute the thread<https://github.com/notifications/unsubscribe/AJ-Fa_yVagwb7zLfc02T8h2LnVmlVVo4ks5qJHRPgaJpZM4IvKwL>.